addons/sale/models/sale_order_line.py

```diff
@@ -60,6 +60,10 @@ class SaleOrderLine(models.Model):
         related='order_id.state',
         string="Order Status",
         copy=False, store=True, precompute=True)
+    tax_country_id = fields.Many2one(
+        comodel_name='res.country',
+        related='order_id.tax_country_id',
+        store=True, precompute=True)
```

Comment:
why is it stored ?

Reply:
I saw that other related fields were stored too, so I assumed there might be a reason to store related fields

Reply:
Only for critical fields that may be/are used in search domains.
Here, I don't think it's necessary ;)